pkg/cmd/server.go

```diff
@@ -76,6 +76,9 @@ func init() {
 	ServerCmd.Flags().String("auth-oidc-client-id", "", "OIDC auth client id")
 	ServerCmd.Flags().String("auth-oidc-client-secret", "", "OIDC auth client secret")
 	ServerCmd.Flags().String("auth-oidc-provider-endpoint", "", "OIDC auth provider endpoint")
+	ServerCmd.Flags().String("auth-oidc-scopes", "", "OIDC requested scopes")
+	ServerCmd.Flags().String("auth-oidc-admin-role", "", "OIDC admin role identifier")
+	ServerCmd.Flags().String("auth-oidc-claim-roles", "", "OIDC claim roles")
```

Comment:
I'm not really familiar with OIDC, but I wonder if the descriptions could be a little clearer -- is this "OIDC claim that communicates roles"?

Reply:
How about description "OIDC claim to inspect for roles" ?